src/deepsparse/loggers/prometheus_logger.py

```diff
@@ -27,24 +28,31 @@
 try:
     from prometheus_client import (
         REGISTRY,
+        CollectorRegistry,
         Summary,
         start_http_server,
         write_to_textfile,
     )
 
     prometheus_import_error = None
 except Exception as prometheus_import_err:
-    REGISTRY = None
-    Summary = None
-    start_http_server = None
-    write_to_textfile = None
+    (
+        REGISTRY,
+        Summary,
+        CollectorRegistry,
+        start_http_server,
+        write_to_textfile,
+    ) = None
```

Comment:
I don't this is valid and will raise an error mostly:
```python
>>> (a, b) = None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: cannot unpack non-iterable NoneType object
```

Reply:
Very tru!